vm-cluster.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: install.istio.io/v1alpha1
```

Comment:
nit?

Reply:
I think you can remove this file as it's just something that is left over on your local machine. Looking it seems to be created as part of https://istio.io/latest/docs/setup/install/virtual-machine/ and we don't actually delete it as part of the cleanup.